sass/components/_chart-controls.scss

```diff
@@ -61,29 +57,37 @@
         padding: 0.35em 2.6em;
         color: #575757;
     }
-    .ciq-menu{
-        cq-menu-dropdown,
-        .cq-menu-dropdown {
-            transform: translateX( -50% ) translateY( 3em );
-            left: 50%;
-            &.cq-menu-dropdown-enter-done, &.cq-menu-dropdown-enter-active{
-                opacity: 1;
-                z-index: 6;
-                transform: translateX( -50% ) translateY( 0em );
-            }
-            &.cq-menu-dropdown-exit-active{
-                opacity: 0;
-                z-index: -3;
-            }
+}
+
+/* Menu dropdown open/close animation */
+.cq-chart-controls .ciq-menu .cq-menu-dropdown {
+    .cq-chart-control-bottom & {
+        /* the left and translateX 50% centers the dropdowns */
+        left: 50%;
+        transform: translateX(-50%) translateY(3em);
```

Comment:
@shayanbinary you have a lot of `left: 50%` and `translateX(-50%)` duplicated all over the scss files. I've consolidated it to only be here.